cmake/pcl_find_cuda.cmake

```diff
@@ -56,8 +54,8 @@ if(CUDA_FOUND)
     set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode arch=compute_${ver},code=compute_${ver}")
     message(STATUS "CMAKE_CUDA_FLAGS: ${CMAKE_CUDA_FLAGS}")
     
-    add_library(pcl_cuda INTERFACE)
-    target_include_directories(pcl_cuda INTERFACE ${CUDA_TOOLKIT_INCLUDE})
-    
   endif ()
+
+  add_library(pcl_cuda INTERFACE)
```

Comment:
@larshg PTAL (since the previous comment regarding empty project was added by you)